src/libsync/capabilities.cpp

```diff
@@ -179,7 +179,9 @@ bool Capabilities::chunkingNg() const
 
 bool Capabilities::userStatus() const
 {
-    return _capabilities.contains("notifications") && _capabilities["notifications"].toMap().contains("user-status");
+    return _capabilities.contains("notifications") &&
```

Comment:
I can't really judge what's on the server but that feels like a strong departure from the structure which was here before. Was it broken before? Why/when did that change?

Mostly being curious here.

Reply:
It was broken, I was not looking for the correct keys. If that is what you mean...

Reply:
Oh OK, fair enough. That explains my confusion. Sorry for the noise.